spring-mvc-simple-2/pom.xml

```diff
@@ -18,6 +18,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I don't think we should be introducing Lombok into the mix for such a trivial model class and where Lombok is not the focus of the article.

Reply:
yeah, sure. I'll remove it.